tests/bug-6207-2/suricata.rules

```diff
@@ -0,0 +1 @@
+alert tcp any any -> any any (msg: "Test file content"; file.data; content:"42"; sid:1;)
```

Comment:
Can we add `filesize: 2` keyword to ensure the file is exactly 42 ant not having 42 as a substring ?